admin/app/components/solidus_admin/orders/show/adjustments/index/component.rb

```diff
@@ -73,10 +73,9 @@ def columns
         header: :adjustable,
         col: { class: 'w-56' },
         data: ->(adjustment) {
-          tag.figure(safe_join([
-            render(component("ui/thumbnail").for(adjustment.adjustable, class: "basis-10")),
-            figcaption_for_adjustable(adjustment),
-          ]), class: "flex items-center gap-2")
+          component_name = adjustment.adjustable&.class&.table_name&.singularize
+          component_key = ["orders/show/adjustments/index/adjustable", component_name].compact.join("/")
```

Comment:
The chain of `&.` is quite long, I wonder if there's a better way to handle it, but if you couldn't find it don't worry.

Reply:
I dry'ed up the method, and now it's just still a long method chain, but not a long maybe chain.